### PR TITLE
fix!: change configuration parameter for buildTarget

### PR DESCRIPTION
**Pull request by dianjuar**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://www.conventionalcommits.org/en/v1.0.0/
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] Both workspaces were tested Angular an Nx

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Chore
- [ ] Refactoring (no functional changes, no api changes)
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

The parameter configuration is migrated to buildTarget.
For migration just change `--configuration` for `--build-target` on the command line and on the configuration file `configuration` for `buildTarget`

### CODE REVIEW

**Comment by dianjuar** [comment]
Closes #60

**Comment by dianjuar** [comment]
FYI @JohannesHoppe

**Comment by JohannesHoppe** [comment]
😎👍

File: src/LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 Diego Juliao, Yossely Mendoza
+Copyright (c) 2020 Diego Juliao, Yossely Mendoza
```
Comment (JohannesHoppe) [inline src/LICENSE:3]: 2019-2021

Comment (dianjuar) [reply src/LICENSE:3]: image

Comment (JohannesHoppe) [reply src/LICENSE:3]: # 🕵️

Comment (dianjuar) [reply src/LICENSE]: Thanks for this review

**Pull request merged**